### DISCUSSION TITLE
ebsd.mis2mean vs grains.innerboundary.misorientation

### DISCUSSION THREAD

**Discussion by amirtopaz** [mtex-toolbox/mtex#641] [Ask Anything]
Hi,
I get two different axis distribution plots when I use the following misorientation commands, can you explain why I get two different plots and what is the difference between the two commands?
I use the following code:
```matlab
[grains,ebsd.grainId] = calcGrains(ebsd('Indexed'),'threshold',[2*degree, 10*degree]);
mori = grains('Hornblende').innerBoundary.misorientation;
mori2 = ebsd('Hornblende').mis2mean;
figure
plotAxisDistribution(mori,'antipodal','smooth');
annotate([Miller(1,0,0,CS{2}),Miller(0,1,0,CS{2}),Miller(0,0,1,CS{2})],...
       'label',{'[100]','[010]','[001]'},'BackgroundColor','w');
mtexColorbar
figure
plotAxisDistribution(mori2,'antipodal','smooth');
annotate([Miller(1,0,0,CS{2}),Miller(0,1,0,CS{2}),Miller(0,0,1,CS{2})],...
       'label',{'[100]','[010]','[001]'},'BackgroundColor','w');
mtexColorbar
```
mori / mori2
![mori](https://user-images.githubusercontent.com/62837798/102712427-f28b3080-42c9-11eb-8253-09191265e4ce.jpg)![mori2](https://user-images.githubusercontent.com/62837798/102712429-f323c700-42c9-11eb-981e-5839eb281393.jpg)

**Comment by ralfHielscher** [comment]
`mori ` is the misorientation at inner boundaries
`mori2` is the misorientation to the grains mean orientation

Ralf
